backend/tests/v5/e2e/routes/teamspaces/index.test.js

```diff
@@ -252,12 +254,12 @@ const testGetTeamspaceMembers = () => {
 			expect(res.body.code).toEqual(templates.teamspaceNotFound.code);
 		});
 
-		test('should return list of users with their jobs with valid access rights', async () => {
+		test('should return list of users with their roles with valid access rights', async () => {
```

Comment:
Can we add test case for:
1. user with no roles (we'd expect an empty array)
2. user with more than one role (we'd expect array length > 1)